packages/backend-modules/auth/lib/AccessToken.js

```diff
@@ -63,6 +63,10 @@ const scopeConfigs = {
     ],
     ttlDays: 5,
   },
+  NOW_YOU_SEE_ME: {
```

Comment:
Verständnisfrage: nur die Session-Tokens werden in der DB abgelegt, oder?

Reply:
Und der Name ist natürlich super :grin:

Reply:
Anmelde-Tokens und Sessions werden in der Datenbank abgelegt.

(Access Tokens sind signierte Tokens, die nur wir verfizieren können. Im Token stecken Informationen drin wie User ID, Scope und Ablaufdatum; können wir einen Token verifizieren, dann erlauben wir bestimmte Dinge.)